backend/app/src/main/java/com/moh/phlat/backend/controller/ControlController.java

```diff
@@ -118,7 +118,7 @@ public ResponseEntity<ResponseMessage> approveToLoadToPLR(@PathVariable("id") lo
 
 			_controlTable.setStatusCode("APPROVED");
 
-			_controlTable.setUpdatedBy("SYSTEM");
```

Comment:
@lysonle let me know if this value "SYSTEM" used at many places will change or remain as "SYSTEM" only

Reply:
All the places where the value is set to "SYSTEM" on the updatedBy and createdBy columns should be replaced by the current login user's IDIR ID.